### ISSUE TITLE
app.css asset not linking correctly on node v5.6.0

### ISSUE THREAD

**Issue by mz3** [iam4x/isomorphic-flux-boilerplate#249]
npm run dev gives a FOUC and the following error in the browser JS console:

```
GET http://192.168.0.16:3001/assets/undefined 404 (Not Found)
```

Is there something off with the `npm run dev` script? Am I missing a global npm dependency?

**Comment by jchapple** [comment]
Hey @mz3. I think the error is related to this commit: https://github.com/iam4x/isomorphic-flux-boilerplate/commit/495132109caa3e347c54ad3b4ee666e16ad8f243, in which `url-loader` is used to handle images/fonts/etc in development mode. It should work that the assets get delivered via JS once Webpack has initialised. At least, that's how it works for me.

As for the FOUC in dev - I think I'm right in saying that this is normal behaviour, and unrelated to the error you mentioned.

Incidentally, @iam4x - I'm wondering if it's possible to have `url-loader` inject the data directly into the `src` attribute for images, for example?

**Comment by utkuturunc** [comment]
Together with this error, I am also getting markup mismatch warning. Using file-loader in dev solves both but then I cannot load fonts from npm modules. I believe 4951321 was done to solve that.

**Comment by mz3** [comment]
You're correct; the assets get delivered once all the client side JS runs. The downside is just the inability to test with JS disabled.

I'm still working to understand the boilerplate so I apologize if my notes are vague or outright incorrect.

**Comment by iam4x** [comment]
Hello guys, I know there's breaking changes with fs module on nodejs5.0.0 haven't tried it yet.

And yes you are right @mz3 it's not possible to test SSR without JS on dev env, would be great to add a command to run boilerplate as production to debug.

**Comment by burgerzzz** [comment]
I had this same problem and it was due to a 'jsx-curly-spacing' lint error, I changed the following in app/components/header.jsx from:

```
        {/* Spinner in the top right corner */}
        {/* LangPicker on the right side */}
        {/* React Logo in header */}
        {/* Links in the navbar */}
```

to:

```
        { /* Spinner in the top right corner */ }
        { /* LangPicker on the right side */ }
        { /* React Logo in header */ }
        { /* Links in the navbar */ }
```

It solved the build error and now assets are loading.

**Comment by dantetwc** [comment]
any update on this?
